packages/transport/src/modules/accounts/IdentityUtil.ts

```diff
@@ -1,56 +1,53 @@
-import { CoreBuffer, CryptoHash, CryptoHashAlgorithm, ICryptoSignaturePublicKey } from "@nmshd/crypto";
-import { CoreAddress, CoreErrors } from "../../core";
+import { CoreBuffer, CryptoHash, CryptoHashAlgorithm, Encoding, ICryptoSignaturePublicKey } from "@nmshd/crypto";
+import { CoreAddress } from "../../core";
 
-export class IdentityUtil {
-    public static async createAddress(publicKey: ICryptoSignaturePublicKey, realm: string): Promise<CoreAddress> {
-        if (realm.length !== 3) throw CoreErrors.general.realmLength();
+const enmeshedPrefix = "did:e:";
```

Comment:
Vague name. How about `enmeshedAddressPrefix`?

Reply:
yeah, i also thought about inlining it completely. As it's only used in this file I don't care about the name that much. will change.

Reply:
I already changed the variable name :D